.travis.yml

```diff
@@ -1,13 +1,16 @@
 language: python
 sudo: false
 python:
+  - "2.6"
   - "2.7"
   - "3.3"
   - "3.4"
   - "3.5"
 install:
   - "pip install pylint"
   - "pip install -r requirements.txt"
+  - "if [ $TRAVIS_PYTHON_VERSION == '2.6' ]; then pip install unittest2; fi"
+  - "sed -i \"s/'ascii', errors='replace'/'ascii', 'replace'/\" $(python -c 'import distutils.sysconfig as s; print(s.get_python_lib())')/potr/proto.py"
 script: ./run_tests.sh
```

Comment:
We can start merging the smaller changes like `{}` to `{0}` to get closer to 2.6 support but I think we should wait until potr supports it before merging the more 2.6-specific changes. I don't want the code to give the impression that it's usable with 2.6 if it isn't yet.